lib/step_functions_stack.py

```diff
@@ -16,10 +16,10 @@
 
 
 from .configuration import (
-    AVAILABILITY_ZONE_1, AVAILABILITY_ZONE_2, AVAILABILITY_ZONE_3, 
+    AVAILABILITY_ZONE_1, AVAILABILITY_ZONE_2, AVAILABILITY_ZONE_3,
     ROUTE_TABLE_1, ROUTE_TABLE_2, ROUTE_TABLE_3,
     S3_RAW_BUCKET, SUBNET_ID_1, SUBNET_ID_2, SUBNET_ID_3, SHARED_SECURITY_GROUP_ID, VPC_ID,
-    get_environment_configuration, get_logical_id_prefix, get_resource_name_prefix, 
+    get_environment_configuration, get_logical_id_prefix, get_resource_name_prefix,
     S3_CONFORMED_BUCKET
```

Comment:
@bwighane what changed here for these files?

Reply:
Formatting, the linter should have removed the extra space at the end of the line.